src/core/BufferGeometry.js

```diff
@@ -663,6 +707,28 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 				}
 
+				// process morph attributes if present
+
+				if ( morphAttributesPosition ) {
+
+					for ( var i = 0, il = morphAttributesPosition.length; i < il; i ++ ) {
+
+						var morphAttribute = morphAttributesPosition[ i ];
+
+						for ( var j = 0, jl = morphAttribute.count; j < jl; j ++ ) {
+
+							vector.x = morphAttribute.getX( i );
+							vector.y = morphAttribute.getY( i );
+							vector.z = morphAttribute.getZ( i );
```

Comment:
maybe this?
```js
vector.fromBufferAttribute( morphAttribute, i );
```

Reply:
Good idea! We can do the same when accessing the `position` attribute 🙌